mnelab/dialogs/xdf_streams.py

```diff
@@ -70,3 +75,6 @@ def effective_srate(self):
     @property
     def prefix_markers(self):
         return self._prefix_markers.isChecked()
+
+    def details(self):
+        self.parent().xdf_meta_info(self.fname)
```

Comment:
Interesting, I never called a method via `parent()`!

Reply:
Yeah, I was just about to create an attribute for `parent` in the `__init__` when I realized it's passed to the base class' `__init__`.